### ISSUE TITLE
can not launch jest

### ISSUE THREAD

**Issue by cuongluu8** [gidztech/jest-puppeteer-docker#36]
I had a simple jest test that uses puppeteer (with `jest-puppeteer` preset) running fine and I wanted to run it within a docker container.
I'm trying to follow the instructions here so basically using `jest-puppeteer-docker` preset instead and adding the example `jest-puppeteer.config.js`.  Now when I launch jest I get the following error:
```
TypeError: Cannot read property 'chromium_revision' of undefined
```

**Comment by testerthomas09** [comment]
me too, any news on this?

**Comment by ivarvolt** [comment]
A quick workaround is in your package.json set cross-env process.env.PUPPETEER_CHROMIUM_REVISION=r768783

Each puppeteer version comes with specific chrome version, you can pick yours from the release:
https://github.com/puppeteer/puppeteer/releases

The fault is on this require, haven't had time to check, put probably it's to do with different package structure or some file not being at root level. 
` 
in setup.js

const revision =
        process.env.PUPPETEER_CHROMIUM_REVISION ||
        process.env.npm_config_puppeteer_chromium_revision ||
        require(path.resolve(puppeteerConfigPath)).puppeteer.chromium_revision;
`

**Comment by cuongluu8** [comment]
Thank you.  This workaround does work.  However since chromium is included in the docker container we should be skipping the download step right?.
I tried setting the environment variable `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` to true so it should bypass this step, but it always seems to require this, so this is a bit confusing.

**Comment by ivarvolt** [comment]
I did it through, `"preinstall": "cross-env process.env.PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true"` in package.json

But this requires that you have cross-env installed beforehand or is globaly accessible.

Also it is an optimization step as I understand, so it's optional. You should be able to get it working with or without it.

**Comment by gidztech** [comment]
Sorry for the very late reply, I don't seem to be getting notifications.

The way it **should** be working is it finds Puppeteer's `package.json` in your `node_modules` folder, and it grabs the supported Chromium version defined, and then uses that to create the Docker container with that version of Chromium setup.

It appears Puppeteer have changed the structure of their `package.json` in a recent version and it now doesn't include the property I rely on. I now need to do a bit more work to grab the version. I will make the change and release a new version today. Thanks for reporting.

**Comment by gidztech** [comment]
@ivarvolt @cuongluu8 @testerthomas09 I've released a new version 1.4.2 which updates the logic for getting the Chromium version from the Puppeteer config. It's backwards compatible with older versions of Puppeteer.

It should be working now but let me know if there are any problems.

**Comment by cuongluu8** [comment]
`PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true` is working now.   Thanks 👍
